src/VisualStudio/Core/Def/Implementation/Library/FindResults/TreeItems/ExternalTreeItem.cs

```diff
@@ -9,14 +9,14 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.Library.FindResults
 {
-    internal class ExternalListItem : AbstractListItem
+    internal class ExternalTreeItem : AbstractTreeItem
```

Comment:
Yes, will comment. I'm also considering renaming it to `ExternalLanguageDefinitionTreeItem`